source/backend/tests/unit/api/Services/ProjectServiceTest.cs

```diff
@@ -208,30 +208,68 @@ public void Get_ProjectById_ShouldFail_NotAuthorized()
             var repository = helper.GetService<Mock<IProjectRepository>>();
 
             // Act
-            Func<Task> act = async () => await service.GetById(It.IsAny<long>());
+            Assert.Throws<NotAuthorizedException>(() => service.GetById(1));
```

Comment:
personally I have a mild preference for using a function here and asserting the throws check in the assert area to get a clean split between act and assert, but interested to understand your preference here.

Reply:
fair, ill update the other tests as well so we get consistency